product_docs/docs/pem/9/pem_online_help/pem_worker_usage_guide.mdx

```diff
@@ -9,251 +9,251 @@ The PEM Worker performs tasks on behalf of the PEM Agent. Please note that all t
-      --server-service-name <service-name> (Database server service name.)
-      --remote-monitoring <true|false> (Allow remote monitoring.)
-      --group <name> (Specify the group name where registered server will be displayed.)
-      --team <database-role> (Specify the name of the database role, on the PEM backend database server, that should have access to the monitored database server.)
-      --owner <name> (Specify the name of the database user who will own the monitored server.)
-      --asb-host-name <hostname> (Host name that the agent will use to connect to the database server. If not provided then '--server-addr' value will be used.)
-      --asb-host-port <port> (Port number that agent will use to connect to database server. If not provided then '--server-port' value will be used.
-      --asb-host-db <database> (Database name that agent will use to connect to database server. If not provided then '--server-database' value will be used.)
-      --asb-host-user <database-user> (Database username that agent will use to connect to database server. If not provided then '--server-user' value will be used.)
-      --asb-ssl-mode <ssl-mode> (Default: 'prefer', SSL mode that agent will use to connect to database server.)
-
-      NOTE:
-       Set environment variable PEM_SERVER_PASSWORD to provide the password for the
-       PEM server to allow the pemworker to connect as a PEM admin user. Set
-       PEM_MONITORED_SERVER_PASSWORD to provide the password of the database server being
-       registered and monitored by pemagent.
-
-    unregister-server-options:
-      --pem-user <username> (Database user, a member of 'pem_admin' role, of the PEM backend database server.)
-      --server-addr (Database server address to unregister from PEM server.)
-      --server-port (Port of database server.)
-      --config-dir <Directory path of agent configuration file (Default: "<pemworker path>/../etc")
-
-    unregister-agent-options:
-      --pem-user <username> (Database user, a member of 'pem_admin' role, of the PEM backend database server.)
-
-    add-webhook-options:
-      --webhook-name (Name of the webhook endpoint.)
-      --webhook-ssl-key (The complete path of the client key file to connect to the webhook server.)
-      --webhook-ssl-crt (The complete path of the client certificate file to connect to the webhook server.)
-      --webhook-ssl-ca-crt (The complete path to the webhook SSL CA certificate file.)
-      --webhook-ssl-crl (The complete path of the CRL file to validate revoked webhook server certificate.)
-      --webhook-allow-insecure-ssl (Default: false, Allow webhooks to call with insecure flag.)
-
-    remove-webhook-options:
-      --webhook-name (Name of the webhook endpoint.)
-
-      NOTE:
-       Set environment variable PEM_SERVER_PASSWORD to provide the password for the
-       PEM server to allow the pemworker to connect as a PEM admin user.
-    ```
+- On Linux:  
+
+```shsession
```

Comment:
lost the indentation - that drops these out of the list. Arguably these should be sections with headings, not list items - but for now they are. Suggest either changing them to headings, or re-adding indentation.